### ISSUE TITLE
Specify output for lgx

### ISSUE THREAD

**Issue by spacek531** [oli414/Blender-RCT-Graphics#32]
I like to keep my object.json and sprites.lgx in a separate dir from the sprite output so I never accidentally bloat the parkobj with the .pngs. Manually copying the "images.dat" folder from the output dir to "sprites.lgx" in the object dir is annoying. I want the tool to build it in the correct place when I click render.

**Comment by oli414** [comment]
I don't believe the parkobj created just includes any files that are in there, but instead only the files that are either specified in the sprites.json, or the lgx file

**Comment by spacek531** [comment]
I realize that. I'm saying that I make them separate and I want the plugin to support me.

**Comment by oli414** [comment]
I don't quite follow what the goal is of doing it manually? Is there something you're doing in between that the add-on doesn't do?

**Comment by spacek531** [comment]
I build objects like this

![image](https://user-images.githubusercontent.com/3454156/224289834-50374c71-9b94-42ac-9e2d-a5c64861ae3c.png)

The goal is to do it automatically. Have the plugin make the sprites.lgx file where I am constructing the object. Using the plugin's parkobj-zip is untenable because generating sprites is too slow.

**Comment by oli414** [comment]
Right okay, makes sense. You need to be able to regenerate the parkobj if you've only made changes to the object.json.

What folder structure would you suggest?

**Comment by spacek531** [comment]
It sounds like the plugin desires a folder structure like this, which is acceptable to me
![image](https://user-images.githubusercontent.com/3454156/224419189-571af44f-4196-4b5b-af3e-932d2ec60dd7.png)
